.github/workflows/cirque.yaml

```diff
@@ -24,6 +24,17 @@ jobs:
 
         runs-on: ubuntu-latest
 
+        container:
+            image: connectedhomeip/chip-build-cirque:0.4.10
```

Comment:
from the latest action failed message, seems like it's not able to found 0.4.10 chip-build-cirque docker image.
I guess you probably want to local generate one and then push to dockerhub.

Reply:
yeah, got ahead of myself a bit